lib/metalsmith.js

```diff
@@ -214,7 +214,8 @@ module.exports = metalsmith(__dirname) // __dirname defined by node.js: name of
       path.normalize('javascripts/application.js'),
       path.normalize('javascripts/head-ie8.js'),
       path.normalize('javascripts/govuk-frontend.js'),
-      path.normalize('javascripts/example.js')
+      path.normalize('javascripts/example.js'),
+      path.normalize('javascripts/vendor/*.js')
```

Comment:
Could we mention somewhere in the PR summary or a commit message that this hashes all files in vendor (not just modernizr)? Just for future archaeology!